src/meta/app/src/schema/table.rs

```diff
@@ -616,3 +616,45 @@ pub struct CountTablesReply {
 pub struct TableIdToName {
     pub table_id: u64,
 }
+
+#[derive(serde::Serialize, serde::Deserialize, Clone, Debug, Eq, PartialEq, Default)]
+pub struct TableCopiedFileNameIdent {
+    pub table_id: u64,
+    pub file: String,
+}
+
+#[derive(serde::Serialize, serde::Deserialize, Clone, Debug, Eq, PartialEq, Default)]
+pub struct TableCopiedFileInfo {
+    pub etag: Option<String>,
+    pub content_length: u64,
+    pub last_modified: Option<DateTime<Utc>>,
+}
+
+#[derive(serde::Serialize, serde::Deserialize, Clone, Debug, PartialEq, Eq)]
+pub struct GetTableCopiedFileReq {
+    pub table: TableNameIdent,
+    pub files: Vec<String>,
+}
+
+#[derive(serde::Serialize, serde::Deserialize, Clone, Debug, PartialEq, Eq)]
+pub struct GetTableCopiedFileReply {
+    pub file_info: BTreeMap<String, TableCopiedFileInfo>,
+}
+
+#[derive(serde::Serialize, serde::Deserialize, Clone, Debug, PartialEq, Eq)]
+pub struct UpsertTableCopiedFileReq {
+    pub table: TableNameIdent,
+    pub file_info: BTreeMap<String, TableCopiedFileInfo>,
+    pub expire_at: Option<u64>,
+}
+
+#[derive(serde::Serialize, serde::Deserialize, Clone, Debug, PartialEq, Eq)]
+pub struct UpsertTableCopiedFileReply {}
+
+#[derive(serde::Serialize, serde::Deserialize, Clone, Debug, PartialEq, Eq)]
+pub struct TruncateTableReq {
+    pub table: TableNameIdent,
+}
```

Comment:
IMHO, truncate-table should use table-id as the identifier.

According to the current implementation of truncate-table, it uses table-id plus a seq to update the table-meta(The same for other XXXReq structs: using table-name as identifier may restrict the calling layer from accessing a specific table by id+ver):
https://github.com/datafuselabs/databend/blob/dae90d856e380ea29716e87148cc69d07ccff8ff/src/query/storages/fuse/src/operations/truncate.rs#L31-L83